tests/integration/ethernet_mtu_test.py

```diff
@@ -26,15 +26,12 @@
 from .testlib import statelib
 from .testlib.statelib import INTERFACES
 
-# FIXME: Once IPv6 disabling is supported, below IPv6 codes should be removed.
-
 
 def test_increase_iface_mtu():
     desired_state = statelib.show_only(('eth1',))
     eth1_desired_state = desired_state[INTERFACES][0]
     eth1_desired_state['state'] = 'up'
     eth1_desired_state['mtu'] = 1900
-    eth1_desired_state['ipv6']['enabled'] = True
```

Comment:
IMHO the test is flawed. Instead of taking the full current state of eth1 and changing some things it should only take the necessary items (name, type) and add the items it would like to change. At least this is what we discussed is needed for users to do for ethernet and IPv6 settings IIRC.

Reply:
Sorry, I do not understand. It is reasonable for the user to get the current state and modify the parts he wants, sending it back. What is wrong about it?
The `edit` command does exactly that.

Reply:
A real user will inspect the full state and see what needs to be changed and setting the state to up with ipv6 disabled would not be allowed with the edit command as well.

Reply:
It is allowed if we follow the workaround based on the latest findings.

Reply:
But it is still not allowed for the regular user since the workaround only happens in the test and not when the user actually used edit. IMHO this make the tests meaningless. Also since it is breaking the mtu tests, it shows that the mtu tests are too broad because they should not test the IPv6 behavior.

Reply:
The "regular" user may have ipv6 disabled in the first place, you do not have control over it one way or the other. We can just say in what conditions it works now.

It breaks other tests because this is an integration test and not unit test, integration tests do break unrelated things. But this last point has nothing to do with this PR anyway.